docker/workers.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:lts-bookworm-slim AS BASE
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+
+FROM base AS pruner
+WORKDIR /app
+COPY . .
+RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install -g turbo
+RUN turbo prune --docker --scope "@workers/feeds" --scope "@workers/invite" --scope "@workers/metadata" --scope "@workers/preferences" --scope "@workers/staff-picks" --scope "@workers/achievements" --scope "@workers/freshdesk" --scope "@workers/leafwatch" --scope "@workers/nft" --scope "@workers/prerender" --scope "@workers/sts" --scope "@workers/ens" --scope "@workers/groups" --scope "@workers/live" --scope "@workers/oembed" --scope "@workers/snapshot-relay"
```

Comment:
`--scope` currently doesn't support a blobs/regex, so workers have to be added manually. Since one also has to enable ports, this is probably fine.